KevinHardy-Cooper/VApp
# Initial FrontEnd

Initial Mock Up for SignIn, SignUp, Dashboard, Settings
Best way to test is to pull the branch locally since there are a lot of template being used (aka a lot of code we didn't write)
How to test:

1. `node src/App.js`
2. start MySQL
3. go to localhost:3000/signin
4. make an account by clicking SignUp
5. sign in using those credentials
6. now you should be /dashboard
7. click "Oauth with twitter"
8. authorize
9. view /settings page with account settings
10. that's it!
